Mathlib/Data/List/Dedup.lean

```diff
@@ -130,7 +130,7 @@ theorem dedup_map_of_injective [DecidableEq β] {f : α → β} (hf : Function.I
 /-- Note that the weaker `List.Subset.dedup_append_left` is proved later. -/
 theorem Subset.dedup_append_right {xs ys : List α} (h : xs ⊆ ys) :
     dedup (xs ++ ys) = dedup ys := by
-  rw [List.dedup_append, Subset.union_eq_right (h.trans <| subset_dedup _)]
+  rw [List.dedup_append, Subset.union_eq_right (List.Subset.trans h <| subset_dedup _)]
```

Comment:
Why is this change needed?

Reply:
Otherwise we get an error that I am unsure how to understand exactly:
```
invalid field notation, function 'trans' does not have argument with type (List.Subset ...) that can be used, it must be explicit or implicit with a unique name
```

But according to the infoview `List.Subset.trans` does take two explicit arguments: `(h₁ : l₁ ⊆ l₂) (h₂ : l₂ ⊆ l₃)`...

Reply:
Do you have a suggestion for how to fix that?